applications/xr_hello_holoscan/CMakeLists.txt

```diff
@@ -29,13 +29,26 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(Eigen3)
 
+# Add nlohmann_json
+FetchContent_Declare(
+  nlohmann_json
+  URL https://github.com/nlohmann/json/releases/download/v3.11.2/json.tar.xz
+  DOWNLOAD_EXTRACT_TIMESTAMP TRUE
+)
+FetchContent_MakeAvailable(nlohmann_json)
+
 find_package(holoscan 2.0 REQUIRED CONFIG)
 
+# Add local XR operators
+add_subdirectory(operators/XrFrameOp)
+add_subdirectory(operators/XrTransformOp)
```

Comment:
Suggest removing in favor of `volume_rendering_xr` impl unless there is a need to fork operator code